### ISSUE TITLE
Option to remove ads?

### ISSUE THREAD

**Issue by AverageHelper** [gorilla-devs/GDLauncher-Carbon#332]
The infobox on "Why are ads needed?" [says](https://github.com/gorilla-devs/GDLauncher-Carbon/blob/6013ab88e69e468f9c706202bc633ab975e0f01f/packages/i18n/locale/english/common.json#L417):

> While ads support the free version of our app, we also offer optional subscriptions for those who prefer an ad-free experience.

I would like to participate in such a subscription and remove ads, ideally without recompiling the launcher myself, but for the life of me I can't find an option in the app or [website](https://gdlauncher.com) to do that! Am I missing something?

**Comment by Ghostblackout** [comment]
I don't want a subscription for a Minecraft launcher, and I have Pi-hole, so the devs are not going to get ads revenue from me. So why can't I just make the ads smaller or just turn them off? Like that whole bar on the side that I can't remove or reshape is insane. Just let me make it smaller or turn it off

**Comment by AverageHelper** [comment]
@Ghostblackout Understandable.

Personally, I have both the means and inclination to pay to remove ads from this launcher, and would like the option to do so. Devs gotta eat, and ppl with (some) funds to spare like I have may want to pitch in, but ads are annoying as hell. I'm sure there's already a fork that's removed the ads if you want to dig around for it.

My issue is that the launcher says there's already an option to pay to remove ads. Where is that option? And if one does not exist, might one be created?

**Comment by blarfoon** [comment]
An option is not yet available, it's currently being blocked by GDL accounts which are WIP. I am currently focusing on bugfixes but will soon resume my work on accounts to unlock this option

**Comment by Amorano** [comment]
so that's a nope? I am going back to GDL 1.2. Thanks!

**Comment by AverageHelper** [comment]
Why closed? The "optional subscriptions" verbiage seems to still exist at https://github.com/gorilla-devs/GDLauncher-Carbon/blob/e048803142f1b8edb024e29329da7dc7be042ca5/packages/i18n/locale/english/common.json#L489...

**Comment by blarfoon** [comment]
I have absolutely no idea why I closed this, as it's still relevant. I am currently working on other things as I work out the details about subscriptions

**Comment by VXsz** [comment]
> subscriptions

C'mon, subscriptions? really?

Look, if there's a one-time payment to remove ads forever, it would be cool and I would pay if it's reasonable since I've used this launcher for a long time and it would be great to support it. But subscriptions are not the way to go, and I doubt anyone wants to pay monthly to a Minecraft launcher so they can have their screen/ui real estate back...

**Comment by ghost** [comment]
garbage organization, fork and move on

**Comment by ClassyCar** [comment]
at this point this launcher is basically just a reskinned curseforge. hell, the ads are even placed in the same locations.

**Comment by ngnlTeto** [comment]
I really like the launcher, could you please make an option to remove ads by paying 10-20$ one time 🙏

**Comment by azurite2** [comment]
It would be really nice to have a one-time payment option to remove ads.

**Comment by staridiot** [comment]
I'm shocked this is still open after two years. Makes me wonder how much they're making from advertisements and telemetry

**Comment by blarfoon** [comment]
Hey, this is somewhat low priority at the moment but I am actively working on it. I can't charge very much only to remove ads, so transactions would very small and be eaten for the most part by transaction costs.
As per the how much we're making from ads, probably not as much as you think, it's enough to keep the company and project running, but subs would definitely drastically increase our revenue, assuming we can find a balance on price.

Something else to consider is that once people pay for something (subs) they have much higher expectations, so you need to be able to provide actual real time support (which I currently can't), so that is also something that needs to be taken into account.


That said, I expect subs to land early next year. I would appreciate it if you could give me an estimate, here or on discord, of a price you would find fair to ONLY remove ads, including what kind of payment it is (one time, recurring, quaterly...).

**Comment by staridiot** [comment]
> That said, I expect subs to land early next year. I would appreciate it if you could give me an estimate, here or on discord, of a price you would find fair to ONLY remove ads, including what kind of payment it is (one time, recurring, quaterly...).

Only removing ads is not what I want (I can only speak for myself, others may just want to remove ads). I see in the privacy settings there's _a lot_ of parties that my data can be sold to. I imagine that brings in more money than the ads. If I'm going to pay for the launcher, it would need to remove the ads **and** the telemetry/data selling. For that I'd probably be willing to pay $5/mo or $60/yr. Not sure one time payments would be feasible since they would need to be quite high to offset the permanent loss of alternate revenue.

**Comment by blarfoon** [comment]
Yes paying would fully remove ads altogether including any kind of data telemetry related to it. Product analytics is a separate thing (which we currently have none of) but will be a separate toggle you won't obviously have to pay for and will either be fully anonymous and opt out or will use a persistent id then it will be opt in
